index.html

```diff
@@ -22,13 +22,14 @@
           </div>
 
           <div class="pre-navigation-links">
-            <h5><a>English</a></h5>
-            <h5><a>My Page</a></h5>
-            <h5><a>Logout</a></h5>
+            <h5><a class="pre">English</a></h5>
+            <h5><a class="pre">My Page</a></h5>
+            <h5><a class="pre">Logout</a></h5>
           </div>
         </div>
       </div>
       <div class="nav">
+				<a href="index.html"><img src="/Capstone-One/imgs/logo-orange.png" class="logo-nav" alt="logoporcilio"></a>
         <div id="hamburger" class="hamburger">
```

Comment:
- [ ] kindly you should fix your mobile menu on scroll because your mobile menu is also being scrolled as other elements. while it should be fixed at the top even if you scroll .and please try to make fill the screen when it's visible .you got this 👍 

when no scroll:

![image](https://user-images.githubusercontent.com/90524466/183122481-56ec1626-2392-4148-aafd-c92a6f4b3a56.png)


when there is scroll:

![image](https://user-images.githubusercontent.com/90524466/183122352-c7f8c825-2a17-494b-8bb8-93c1590e00e6.png)